cmd/kpod/create.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"fmt"
+
+	spec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	pb "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
+)
+
+type createResourceConfig struct {
+	blkioWeight       int64
+	blkioDevice       []string
+	cpuShares         int64
+	cpuCount          int64
+	cpuPeriod         int64
+	cpusetCpus        string
+	cpusetNames       string
+	cpuFile           string
+	cpuMems           string
+	cpuQuota          int64
+	cpuRtPeriod       int64
+	cpuRtRuntime      int64
+	cpus              int64
+	deviceReadBps     []string
+	deviceReadIops    []string
+	deviceWriteBps    []string
+	deviceWriteIops   []string
+	memory            string
+	memoryReservation string
+	memorySwap        string
+	memorySwapiness   string
+	kernelMemory      string
+	oomScoreAdj       string
+	pidsLimit         string
+	shmSize           string
+	ulimit            []string
+}
+
+type createConfig struct {
+	additionalGroups []int64
```

Comment:
What is the correlating command line switch for this?  There is a groupAdd below as well.

Reply:
Nice catch, removing.